src/styles/variables.css

```diff
@@ -10,7 +11,13 @@
 	--rounded-corners: 2px;
 	--short-animation-duration: .2s;
 
+	--block-spacing: 25px;
+	--line-spacing: 5px;
+
 	--title-bar-bg: var(--dojo-blue);
 	--title-bar-color: var(--dojo-white);
 	--underlay-bg: var(--dojo-shade-gray);
+
+	--focus-highlight: 0 0 4px 1px var(--dojo-light-blue);
+	--input-padding: 0.5em 1em;
```

Comment:
Whilst consistency across our codebase is valid, we need to have the age old `px vs em vs rem` debate - for which I've added: #35